docs/gemstash-private-gems.7.md

```diff
@@ -81,8 +80,7 @@ If your Gemstash server isn’t running, go ahead and start it:
 
     $ gemstash start
 
-Push your test gem using
-    Rubygems:
```

Comment:
I just tried to release a new Gemstash version and noticed our `rake doc` task is generating the docs in this style 🤔

Reply:
IIRC, there were version incompatible details re Pandoc, so I did all I could to have _something_ run.

Perhaps we can make improvements, now.